getnacelle/nuxt-reference-store
# feat(pdp): adds product page

### WHY are these changes introduced?

Creates PDP [[LD-1106](https://nacelle.atlassian.net/browse/LD-1106)] 

### WHAT is this pull request doing?

- Adds Product page
- Adds Related Products section

### How to Test

1. Run `npm install` to install Tailwind plugins.
2. Run `npm run dev`.
3. Navigate to `/products/aurora-high-tops` to see variant selection, multiple images, features, and related products.
4. Navigate to any other product to test.
5. CMS updates can be made with the following PR: https://github.com/getnacelle/nacelle-reference-store-sanity-studio/pull/5
